dist/stat_data.py

```diff
@@ -444,6 +450,8 @@ def __init__(self, name, desc, flags=''):
     EvictStat('eviction_walks_stopped', 'eviction walks gave up because they restarted their walk twice'),
     EvictStat('eviction_worker_evict_attempt', 'evict page attempts by eviction worker threads'),
     EvictStat('eviction_worker_evict_fail', 'evict page failures by eviction worker threads'),
+    EvictStat('eviction_worker_max_evict_fail', 'maximum evict page failures by eviction worker threads'),
```

Comment:
I think it is not useful for this stat. May be the checkpoint version is enough?